lib/master.js

```diff
@@ -257,7 +257,8 @@ Master.prototype._startWorkers = function(remainingWorkers, res) {
                         worker.removeListener('exit', workerExit);
                         self._currentStartingWorker = undefined;
                         self._firstWorkerStarted = true;
-                        res.push(msg.serviceReturns[0]);
+                        res.push(msg.serviceReturns.length > 1 ?
+                            msg.serviceReturns : msg.serviceReturns[0]);
```

Comment:
The README says,

> // startupResults is an array of arrays of objects returned by each service

so this should really just be `msg.serviceReturns`.  Do you prefer that?